esp8266-fastled-webserver/include/Fields.hpp

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
SIMPLE FILE REVIEW

This header corresponds to Fields.cpp, and exposes only the portions that are used externally.